### DISCUSSION TITLE
Example of changing destination port for XDP programs?

### DISCUSSION THREAD

**Discussion by bithamr** [redsift/redbpf#346] [Q&A] [answered]
Looking around the repository I didn't see much in the way of XDP examples.

In particular I'm looking for information on how to change destination port (for example when doing TCP/UDP load-balancing).

Any help or context on how to do this would be appreciated, thank you!

**Comment by shaneutt** [comment, marked as answer]
I've added example code for this in https://github.com/foniod/redbpf/pull/347
